lib/ansible/module_utils/network/nxos/nxos.py

```diff
@@ -371,17 +376,39 @@ def _send(commands, output):
 
         return responses
 
-    def load_config(self, commands, return_error=False, opts=None):
+    def load_config(self, commands, return_error=False, opts=None, replace=None):
         """Sends the ordered set of commands to the device
         """
+        if replace:
+            commands = 'config replace {0}'.format(replace)
```

Comment:
`config replace <filename>` command works different on nxos than other platforms. Here `replace` has to be the path to the file instead of an option like `config` or `block` blah blah. 
https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/network/nxos/nxos_config.py#L308
Instead of taking this in load_config, we should leave it in the module itself and use `replace_src` param there instead of `replace`.